states/state.py

```diff
@@ -18,6 +18,8 @@ def list_options(cls: Type[T]) -> List[str]:
 
 
 class State:
+    """Function for main state and error catcher"""
```

Comment:
This class is used in multiple places, but never on its own. Specifically, it's extended off of. So it might be best described as 'Base class for game states' and maybe tack on a bit about what functions it provides.

Reply:
Changed to `Base class for game states. This receives user input and will run application accordingly even in the case an option not available is chosen.`